benchmarks/config/base/base.json

```diff
@@ -89,6 +89,13 @@
       "flags": [ "-n", "100" ],
       "extensions": [ "(avx2|asimd)" ]
     },
+    "fp32_fully_connected_mlir": {
+      "type": "MLIR",
+      "benchmark": "fully-connected-fp32-1layer.mlir",
```

Comment:
Please call it `fc-fp32...` to avoid very long names, and make it 3 layers, so that it can be compared to others.